tests/lib/rules/key-spacing.js

```diff
@@ -757,6 +757,34 @@ ruleTester.run("key-spacing", rule, {
             }
         }],
         parserOptions: { ecmaVersion: 6 }
+    }, {
+        code: [
+            "var obj = {",
+            "    foo : 1, 'bar' : 2, baz : 3, longlonglong : 4",
```

Comment:
Can we add more varied tests? Some examples:

```
var obj = {   foo : 1, 'bar' : 2, baz : 3, longlonglong : 4 };
```

```
var obj = {  
    foo : 1, 
    'bar' : 2, baz : 3, longlonglong : 4 
};
```

```
var obj = {  
    foo : 1, 
    'bar' : 2, baz : 3, 
    longlonglong : 4 
};
```

Reply:
@kaicataldo 
Yes, But I have a question.
This PR does not have differences with the actual version([demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgXCJrZXktc3BhY2luZ1wiOiBbXCJ3YXJuXCIsIHtcIm11bHRpTGluZVwiOiB7XCJhbGlnblwiOiBcImNvbG9uXCJ9fV0qL1xuXG5zdHlsZT17XG4gZm9vOiAxLCBiYXI6IDIsXG4gbG9uZ2xvbmdsb25nOiAzLFxufVxuIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoxMCwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6eyJqc3giOnRydWV9fSwicnVsZXMiOnt9LCJlbnYiOnt9fX0=)) in some cases you mentioned.
```
var obj = {  
    foo : 1, 
    'bar' : 2, baz : 3, longlonglong : 4 
};
```
I already opened PR(#12651) for those cases but closed it, because the closed PR would change the concept of the "group" in this rule.
```
// closed one 
var obj = {  
    foo : 1,  // groupA
    'bar' : 2, baz : 3, longlonglong : 4  // groupB
};
```
so I want to know which behavior is correct ,This PR or closed PR(#12651)?

Reply:
I think the behavior here is correct.

Reply:
My intention in requesting tests here is to protect against regressions. Does that make sense?

Reply:
yes! That makes sense. I'll work on it.

Reply:
@kaicataldo 
all done! thanks! :)

Reply:
Thank you!